src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,64 @@
+.calendar {
+  width: $calendar-width;
+  margin: 0 auto;
+  padding: $calendar-padding;
+  display: flex;
+  gap: $calendar-gap-size;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+
+  &__day {
+    display: none;
+    position: relative;
+    width: $day-block-size;
+    height: $day-block-size;
+    background-color: #eee;
+    border: 1px solid black;
+    box-sizing: border-box;
+    cursor: pointer;
+    transition: 0.5s;
+  }
```

Comment:
By default transition is used for all. It's pretty hard for browser - better set only those properties you need
```suggestion
    cursor: pointer;
    transition: transform 0.5s, background-color 0.5s;
  }
```